src/info.h

```diff
@@ -229,7 +229,7 @@ void A_NapalmScatter();
 void A_SpawnFreshCopy();
 
 // ratio of states to sprites to mobj types is roughly 6 : 1 : 1
-#define NUMMOBJFREESLOTS 1024
+#define NUMMOBJFREESLOTS 2048
```

Comment:
Probably better keep old value to keep compatibility with vanilla clients (so, there would be no situation when server runs fine but clients crash with "too much freeslots" error or like that)